docs/source/topologies_openstack.rst

```diff
@@ -245,6 +245,66 @@ Openstack Full Stack
         test_var2: "test_var2 msg is grp2 hello"
         test_var3: "test_var3 msg is grp2 hello"
 
+
+Steps to provision Single Host
+==============================
+
+.. contents:: Topics
+
+.. _Steps to provision Single Host:
+
+Credentials
+```````````
+   * save openstack credentials in ```$WORKSPACE/credentials/clouds.yml``` file using below format
```

Comment:
The path to credentials is variable, as you may have understood from below where you passed `--creds-path`. Consider instead of `$WORKSPACE/credentials/clouds.yml`, using something more generic. Like `/path/to/cloud_credentials.yml`.  Additionally, this setting is optional. Meaning the user could instead use the standard clouds.yaml location, or set the env variable `$CREDS_PATH`. I think it would be very helpful to point out that this is optional.